source/renderer/app/components/widgets/DialogFullSizeOverride.scss

```diff
@@ -19,9 +19,9 @@
     .AutocompleteOverrides_autocompleteContent,
     .SimpleInput_customValueBlock,
     .SimpleInput_customValueWrapper,
-    .SimpleInput_input,
-    .SimpleInput_input:focus,
-    .SimpleInput_input:hover,
+    .SimpleInput_input:not(.SimpleInput_disabled),
```

Comment:
I couldn't find any other solution to get around the declaration with `!important` flag without doing a comprehensive refactoring.